python3/README.md

```diff
@@ -0,0 +1,10 @@
+# Python3
+
+Before running this test code, make sure you have your `api_key` set as the environment variable, `FORWARD_FINANCING_API_KEY`.
+
+Also, ensure that you have [requests](http://docs.python-requests.org/en/master/) installed as that is how we will be making web requests.
```

Comment:
a `Requirements` header may be nice here?

`Also, ensure that you have [requests](http://docs.python-requests.org/en/master/) installed as that is how we will be making web requests.` Maybe something along the lines of:
`Also ensure to/Please also install the [requests](http://docs.python-requests.org/en/master/) library.`